aptible/database.go

```diff
@@ -26,167 +28,139 @@ type DBCreateAttrs struct {
 	DiskSize      int64
 }
 
-func (c *Client) CreateDatabase(env_id int64, attrs DBCreateAttrs) (Database, error) {
+func (c *Client) CreateDatabase(accountId int64, attrs DBCreateAttrs) (Database, error) {
```

Comment:
The formal go style for this is accountID based on the [official style guide](https://github.com/golang/go/wiki/CodeReviewComments#initialisms). I tend to follow it for consistency sake but I'm not super bothered about it if you like `Id` better.

Reply:
:+1: that's fine. I'll update to that. I thought I was following in Terraform's steps here, but on reflection I was not, so no idea what I was thinking